src/store/ChartState.js

```diff
@@ -163,6 +165,20 @@ class ChartState {
             this.comparisonStore.removeAll();
         }
 
+        if (crosshairState !== null && crosshairState !== this.crosshairState) {
+            this.mainStore.crosshair.onChange(crosshairState);
+            this.mainStore.state.saveLayout();
```

Comment:
@shayan-binary-pr this line and the line below both are duplicated while there are in `onChange` function.